doc/man/8/ceph.rst

```diff
@@ -482,6 +482,31 @@ Usage::
 
 	ceph osd create {<uuid>} {<id>}
 
+Subcommand ``new`` reuses a previously destroyed OSD *id*. The new OSD will
+have the specified *uuid*, and the command expects a JSON file containing
+the base64 cephx key for auth entity *client.osd.<id>*, as well as optional
+base64 cepx key for dm-crypt lockbox access and a dm-crypt key. Specifying
+a dm-crypt requires specifying the accompanying lockbox cephx key.
+
+Usage::
+
+    ceph osd new {<id>} {<uuid>} -i {<secrets.json>}
+
+The secrets JSON file is expected to maintain a form of the following format::
```

Comment:
this JSON passing is new right? or are there other CLI endpoints that can consume JSON? From an API standpoint it might be easier to allow flags vs. JSON.